workflows/Blacklist_Indicators_with_Microsoft_Defender_ATP_from_Microsoft_Teams/workflow.spec.yaml

```
@@ -0,0 +1,27 @@
+extension: workflow
+products: ["insightconnect"]
+name: Blacklist_Indicators_with_Microsoft_Defender_ATP_from_Microsoft_Teams
+title: Blacklist Indicators with Microsoft Defender ATP from Microsoft Teams
+description: Use the simplicity of Microsoft Teams to blacklist and unblacklist indicators such as hashes, domains, URLs, and IP addresses with Microsoft Defender ATP.
```

```suggestion
description: Blacklist and unblacklist indicators such as hashes, domains, URLs, and IP addresses with Microsoft Defender ATP from Microsoft Teams.
```

Shorter so it fits in the Extension Library listing a bit better